events/tests/test_event_images_v1.py

```diff
@@ -217,6 +217,82 @@ def test__upload_an_image_with_api_key(
     assert image.format == "PNG"
 
 
+@pytest.mark.django_db
+def test__image_edit_as_superuser(
+    api_client, list_url, image_data, user, organization, organization2, user2
+):
+    organization.admin_users.add(user)
+    api_client.force_authenticate(user)
+
+    response = api_client.post(list_url, image_data)
+
+    user2.is_superuser = True
+    organization2.admin_users.add(user2)
+    api_client.force_authenticate(user2)
+    detail_url = reverse("image-detail", kwargs={"pk": response.data["id"]})
+    response2 = api_client.put(detail_url, {"name": "this is needed"})
+    assert response2.status_code == 200
+
+
+@pytest.mark.django_db
+def test__image_edit_as_external(
+    api_client, settings, list_url, image_data, user, organization
+):
+    organization.admin_users.add(user)
```

Comment:
External users shouldn't be admins in any organizations, since external users are users that do not belong to any organization.

Reply:
Changed this so that they are not admins.